spec/services/scrapers/parties/high_priority.rb

```diff
@@ -13,6 +13,15 @@
         end
       end
     end
+    context 'when there is a party with no html and no oscn_id' do
+      let!(:party_no_oscn) { create(:party, oscn_id: nil) }
+
+      it 'doesnt add jobs to the PartyWorker' do
```

Comment:
instead of "doesn't" I typically use "does not" (or else I think you I don't have to use double quotes or a \ to escape the single quote)